app/pages/lab-pages-editor/components/TasksPage/TasksPage.jsx

```diff
@@ -140,7 +140,8 @@ export default function TasksPage() {
     update({ tasks: newTasks });
   }
 
-  const previewUrl = 'https://frontend.preview.zooniverse.org/projects/darkeshard/example-1982/classify/workflow/3711?env=staging';
+  const previewEnv = getPreviewEnv();
+  const previewUrl = `https://frontend.preview.zooniverse.org/projects/${project?.slug}/classify/workflow/${workflow?.id}${previewEnv}`;
```

Comment:
Should we maybe do something different here if there isn't a project slug and workflow id? Because if its missing and it gets clicked on I would rather it send them somewhere predictable/debuggable instead of a default weird url with null/undefined/ect in it.

Reply:
Let me think about this. 🤔 

You're right that a URL like `https://frontend.preview.zooniverse.org/projects/undefined/classify/workflow/undefined` would be pretty wonky. But it's a fairly safe assumption every project has a slug and every workflow has an ID. But _but_ it's still good to have a safety for when those assumptions bork.